README.md

```diff
@@ -1,59 +1,170 @@
-# Qᴜᴀʟᴛʀᴀɴ
+<!-- H1 title omitted because our logo acts as the title. -->
```

Comment:
maybe put the title in this comment? idk folks are liable to read this file from vim or something after they've checked out the repo

Reply:
Good idea!